docs/html/explanations/authentication.md

```diff
@@ -0,0 +1,86 @@
+# Authentication
+
+## Basic HTTP authentication
+
+pip supports basic HTTP-based authentication credentials. This is done by
+providing the username (and optionally password) in the URL:
+
+```
+https://username:password@pypi.company.com/simple
+```
+
+For indexes that only require single-part authentication tokens, provide the
+token as the "username" and do not provide a password:
+
+```
+https://0123456789abcdef@pypi.company.com/simple
+```
+
+### Percent-encoding special characters
+
+Certain special characters are not valid in the credential part of a URL.
+If the user or password part of your login credentials contain any of these
+[special characters][reserved-chars], then they must be percent-encoded. As an
+example, for a user with username `user` and password `he//o` accessing a
+repository at `pypi.company.com/simple`, the URL with credentials would look
+like:
+
+```
+https://user:he%2F%2Fo@pypi.company.com/simple
+```
+
+[reserved-chars]: https://en.wikipedia.org/wiki/Percent-encoding#Percent-encoding_reserved_characters
+
+```{important}
+Support for percent-encoded authentication in index URLs was added in pip 10.0
+(in {pull}`3236`). Users with older versions of pip need to upgrade to a newer
+version of pip, to be able to use characters that need to be percent-encoded.
+```
+
+## netrc support
+
+pip supports loading credentials from a user's `.netrc` file. If no credentials
+are part of the URL, pip will attempt to get authentication credentials for the
+URL's hostname from the user's `.netrc` file. This behaviour comes from the
+underlying use of [`requests`][pypi-requests], which in turn delegates it to
```

Comment:
This seems broken

![Screenshot 2021-02-24 at 18 51 43](https://pip--9474.org.readthedocs.build/en/9474/explanations/authentication/)

Reply:
Ah yes, I never added the link. :)